apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -18,10 +19,12 @@ const Header = (): JSX.Element => {
 
 	return (
 		<header className={styles["header"]}>
-			<div className={styles["logo-container"]}>
-				<img alt="GitFit logo" className={styles["logo-img"]} src={logoSrc} />
-				<span className={styles["logo-text"]}>Logo</span>
-			</div>
+			<NavLink className={styles["logo-link"] ?? ""} to={AppRoute.ROOT}>
```

Comment:
Maybe let's stick to the following approach
```suggestion
			<NavLink className={styles["logo-link"] as string} to={AppRoute.ROOT}>
```
Currently, in our project we have both `?? ""` and `as string`, so we should decide, which one to stick with. And `as string` looks more prefferable cause we know for sure, that this style is a string.

@liza-veis what do you think?

Reply:
agreed, it will be better